test/Tests/src/Data/Vector_Spec.enso

```diff
@@ -212,6 +212,15 @@ spec = Test.group "Vectors" <|
         concat = [1, 2, 3] + [4, 5, 6]
         concat.should_equal [1, 2, 3, 4, 5, 6]
 
+    Test.specify "Vector slice should return a Vector" <|
+        vec = [1, 2, 3, 4, 5, 6]
+        vec.slice 0 3 . should_equal [1, 2, 3]
+        vec.slice 1 3 . should_equal [2, 3]
+        vec.slice 1 1 . should_equal []
+        vec.slice 0 100 . should_equal [1, 2, 3, 4, 5, 6]
```

Comment:
```suggestion
        vec.slice 0 100 . should_equal vec
```

Reply:
Radek, do you suggest to test for identity? Then the check has to use `Meta.is_same_object`. Otherwise `==` on _Arrays&Vectors_ always only compares content.

Reply:
No, I suggest the check on content, I just don't see a reason for writing `[1, 2, 3, 4, 5, 6]` twice if its exactly the content of `vec`. But I get your point that this could _falsely_ suggest a check on _identity_ so probably good to leave it as-is.